src/analytics/Properties.tsx

```diff
@@ -152,6 +153,11 @@ interface AppEventsProperties {
     fullPath: string | null
     query: string | null
   }
+  [AppEvents.show_gas_fee_warning]: {
+    flow: GasFeeWarningFlow
+    errorType: 'need-decrease-spend-amount-for-gas' | 'not-enough-balance-for-gas'
+    tokenNeeded: string
```

Comment:
nit: `tokenId` should be fine here for analytics and if we need more info in the key it appears more commonly prefixed.

```suggestion
    tokenId: string
```

Reply:
addressed in [7a33d2b](https://github.com/valora-inc/wallet/pull/6385/commits/7a33d2b785245904a61f8f91870a3334c52aa935)